src/main/java/net/masterthought/cucumber/ReportInformation.java

```diff
@@ -1,11 +1,6 @@
 package net.masterthought.cucumber;
 
-import java.text.SimpleDateFormat;
-import java.util.ArrayList;
-import java.util.Date;
-import java.util.List;
-import java.util.Map;
-
+import com.googlecode.totallylazy.Sequence;
```

Comment:
May I know is there any significance for the import order? Is there any specific reason or is it a best practice to avoid changes to import statements?